src/App.js

```diff
@@ -14,6 +14,7 @@ import Products from './screens/Products';
 import DailyGrow from './screens/DailyGrow';
 import Consulting from './screens/Dashboard';
 import GreenHouse from './screens/GreenHouse';
+import ErrorScreen from './screens/ErrorScreen';
```

Comment:
Just like others, please rename it as Error only instead of ErrorScreen

Reply:
Will do. So I make the changes and make another commit?

Reply:
Yes